src/project_creator/templates/project/{{cookiecutter.name}}/{% if cookiecutter.touch_components -%} components {%- endif %}/component/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 spl_add_source(src/component.c)
 spl_add_test_source(test/test_component.cc)
-spl_create_component()
+spl_create_component(LONG_NAME "Component")

```

Comment:
We need to define long names to be used in the documentation. Without this, the `component_name` was used, which is just the component path with the slashes replaced by underscores.